test/services/dumpers/agent-nodejs/expected/javascript/mariadb/package.json

```diff
@@ -1,11 +1,10 @@
 {
-  "name": "test-output-mariadb",
+  "name": "test-output-javascript-mariadb",
   "version": "0.0.1",
   "private": true,
-  "main": "index.js",
   "scripts": {
-    "start:watch": "nodemon",
-    "start": "node ./index.js"
+    "start": "nodemon ./index.js",
+    "start:agent": "node ./index.js"
   },
```

Comment:
The start command will start the nodemon script, and the `start:agent` command is the classic `node` script.
This match the scripts from `agent-nodejs/packages/_exemple` and it will allow to do the same for typescript, which is perfect since it will allow users to start the agent without needing to build it before (we're going to use `ts-node`).
Validated by @romain-gilliotte :)

Reply:
Hummmm question. Is it written somewhere that people should use start:agent in production ?